.gitignore

```diff
@@ -27,3 +27,4 @@ data/payloads/*
 !data/payloads/.gitkeep
 data/facts/*yml
 !data/facts/.gitkeep
+conf/default.yml
```

Comment:
I don't think we want this line ^^ we want the default.yml to stay in the repo. other .yml files in conf, probably not though...

Reply:
Yes I know.... I was trying to find a way to ignore that commit =/